nixos/modules/services/misc/memos.nix

```
@@ -0,0 +1,192 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+let
+  cfg = config.services.memos;
+in
+{
+  options.services.memos = {
+    enable = lib.mkEnableOption "Memos note-taking";
+    package = lib.mkPackageOption pkgs "memos" { };
```

Tip: **if** you want "Memos" capitalised to match your `enable` option's description, you can do:

```suggestion
    package = lib.mkPackageOption pkgs "Memos" {
      default = "memos";
    };
```